src/streamlink/plugins/ustreamtv.py

```diff
@@ -79,11 +80,28 @@ def cluster(self, cluster):
         self._cluster = cluster
         self.reconnect()
 
+    @staticmethod
+    def parse_proxy_url(purl):
+        proxy_options = {}
+        if purl:
+            p = urlparse(purl)
+            if p.scheme != "http":
```

Comment:
socks5 seems supported

https://github.com/websocket-client/websocket-client/pull/417

Reply:
Ah, I didn’t see that - in the docs for the function it says http only.

Reply:
Updated it so that it'll pass it through to the websocket library to handle.